src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -145,6 +149,34 @@ class CollapsibleCourseRun extends React.Component {
     this.setCollapsible(true);
   }
 
+  displayOfacRestriction(restriction) {
+    switch (restriction) {
+      case false:
+        return 'No';
+      case true:
+        return 'Yes';
+      default:
+        return '--';
+    }
+  }
+
+  displayDefaultButtonLabel(status, isAdmin) {
+    if (isAdmin) {
+      switch (status) {
+        case REVIEW_BY_LEGAL:
+          return 'Save & Send to PC Review';
```

Comment:
this line is only viewed by staff members, never regular users.  line 177 is for regular users.  i  dont think the language has to match because the the labels are for 2 totally separate flows.  Also, this is what is on the mock.